### PR TITLE
Switch to using scoped instances instead of singletons

### PR DESCRIPTION
**Pull request by AlexVanderbist**
Using singletons for settings classes can cause the settings instance to remain in the IOC container over multiple jobs (Horizon) or multiple requests (Octane). This isn't an issue for most projects, as settings don't randomly change every request. However, in a multi-tenancy environment the loaded settings different every job and every request, based on the tenant. In this case we do not want to re-use a settings instance that's still around in the IOC container from the previous request.

TL;DR: switching `singleton` to `scoped` causes the settings class instances to be refreshed every request & every job

https://laravel.com/docs/8.x/container#binding-scoped

### CODE REVIEW

**Comment by rubenvanassche** [comment]
Merci!

**Comment by atapatel** [comment]
@AlexVanderbist Man, hats off to you for this solution.
